### DISCUSSION TITLE
MAC address

### DISCUSSION THREAD

**Discussion by 26zl** [HotCakeX/Harden-Windows-Security#380] [General]
Hi! Is their any possible way to force daily change of the mac address on every single wifi/ethernet i connect too? Without going in and manually doing it? Is their maybe a gpo for this? 
Thanks in advance

**Comment by HotCakeX** [comment]
Hi,
There is no GPO at the moment, and i couldn't find the registry key that is responsible for it. I'll try again to see if i'll have better luck this time

**Reply by 26zl** [reply]
> Hi,
> 
> There is no GPO at the moment, and i couldn't find the registry key that is responsible for it. I'll try again to see if i'll have better luck this time

I see! No worries. Thanks anyway! Another question, if you don’t mind. I remember you mentioned some time ago that you use Mullvad VPN. Do you still use it? And is there a possibility it could leak information based on the settings from your program? I use Sandbox quite often — could it leak when I’m using the “local network sharing” setting in Mullvad? And do you know if their are any downsides using that specific setting on mullvad when talking about general network security?

**Reply by HotCakeX** [reply]
Sorry for late reply, totally missed it!

Yes, i still use it sometimes, mostly when i'm not using my own home router. Local network sharing is necessary so VMs such as Windows Sandbox or Hyper-V's default switch can have internet connectivity.

Windows Sandbox is alright but if you want even more control then setting up a Hyper-V VM is even better and reduces the possibility of information leakage even more.

For example you can use Mullvad on your host, then install Cloudflare WARP on your VM and then use the Internet. Also Windows Firewall plus the configurations applied to it by the Harden Windows Security module, greatly reduces possibly of information leakage.

If you're talking about websites getting information about your system using JavaScript, then you need to make your VM as different as possible from your host. E.g, installed extensions, installed fonts, installed system language, system timezone, browser program, amount of RAM, screen resolution/size and so on.

**Reply by 26zl** [reply]
Thank you so much for taking the time to write such a detailed and helpful response. I really appreciate all the advice you’ve shared. The tips about using VMs and setting up security configurations, especially with Mullvad and Cloudfare WARP, are super useful. I’ll definitely look into setting up a Hyper-V VM for more control and try some of the other suggestions you mentioned. Thanks again for all your help, you are always helpful!

**Reply by HotCakeX** [reply]
You're very welcome, don't mention it ^^
